tailwind.config.js

```diff
@@ -1,17 +1,21 @@
 module.exports = {
-  mode: 'jit',
-  purge: ['./public/**/*.html', './src/**/*.{astro,js,jsx,svelte,ts,tsx,vue}'],
+  mode: "jit",
+  purge: ["./public/**/*.html", "./src/**/*.{astro,js,jsx,svelte,ts,tsx,vue}"],
+  darkMode: "class",
   theme: {
     extend: {
+      colors: {
+        darkSecondary: "#151414",
```

Comment:
Codacy has a fix for the issue: [Unexpected trailing comma.](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8426368)
```suggestion
        darkSecondary: "#151414"
```